bin/install-docker.sh

```diff
@@ -31,10 +31,15 @@ echo -e $(status_message "Starting Docker containers...")
 docker-compose up -d >/dev/null
 
 HOST_PORT=$(docker-compose port wordpress 80 | awk -F : '{printf $2}')
+PROTOCOL='http'
+if [ "$1" == '--ssl' ]; then
+	HOST_PORT=$(docker-compose port wordpress 443 | awk -F : '{printf $2}')
+	PROTOCOL='https'
+fi
 
 # Wait until the docker containers are setup properely
 echo -en $(status_message "Attempting to connect to wordpress...")
-until $(curl -L http://localhost:$HOST_PORT -so - 2>&1 | grep -q "WordPress"); do
+until $(curl -L --insecure $PROTOCOL://localhost:$HOST_PORT -so - 2>&1 | grep -q "WordPress"); do
```

Comment:
In the future we should probably allow users to specify their hostname; I, for instance, find it handy to assign my Mac's local network name here instead of `localhost` for easier mobile/VM testing.